pep-513.rst

```diff
@@ -54,8 +54,8 @@ in practice, is compatible enough that packages conforming to this standard
 will work on *many* linux systems, including essentially all of the desktop
 and server distributions in common use. We know this because there are
 companies who have been distributing such widely-portable pre-compiled Python
-extension modules for Linux -- e.g. Enthought with Canopy [2]_ and Continuum
-Analytics with Anaconda [3]_.
+extension modules for Linux -- e.g. Enthought with Canopy [4]_ and Continuum
+Analytics with Anaconda [5]_.
```

Comment:
FYI for future reference, I don't think anyone cares about whether end note numbers are assigned sequentially in PEPs :-)